doc/source/development/packaging.rst

```diff
@@ -0,0 +1,98 @@
+==================
+Packaging releases
+==================
+
+OpenGHG is now completely tested and deployed using GitHub actions.
+The most updated developer code lies in the `OpenGHG repository <https://github.com/openghg/openghg>`_
+
+The development process should be:
+
+* New changes to any section of the code base should always be made using feature branches.
+
+.. code-block:: bash
+
+   git checkout -b branch_name
+
+.. note::
+   branch_name - Iss(Issue number) recommended if the change is a bug fix for an issue. E.g. “Iss123_add_attributes”. May also want to include “fix_” and “feature_” as appropriate.
+
+* Pull requests are issued from these branches to ``devel``. All merge conflicts
+  must be fixed in the branch and all tests must pass before the pull
+  request can be merged into ``devel``.
+
+The result of this is that ``devel`` should contain the fully working 
+tested, and most up-to-date version of ``OpenGHG``. However, this
+version should not be used for production runs.
+
+Defining a release
+------------------
+
+Releases aim to be backwards compatible and capable of being used for production runs, at least for
+the functionality that is fully described in the tutorial.
+
+We use `semantic versioning <https://semver.org>`__ and take care
+not to cause breaking changes in the public API.
```

Comment:
The semvar website says we should not have version numbers starting with 0. I guess it's not the most important part of what they specify

Reply:
Yes, we can decide on it sometime